.circleci/config.yml

```
@@ -13,9 +13,9 @@ jobs:
     parameters:
       stack-version:
         type: enum
-        enum: ["18", "20"]
+        enum: ["18", "20", "22"]
     machine:
-      image: ubuntu-2004:202107-02
+      image: ubuntu-2004:202111-02
```

Latest tag is `202201-02`:
https://circleci.com/developer/machine/image/ubuntu-2004#image-tags

```suggestion
      image: ubuntu-2004:202201-02
```

---

(I presume you copied the version from https://circleci.com/docs/2.0/configuration-reference/#available-machine-images; unfortunatelty those docs are outdated, I've filed https://github.com/circleci/circleci-docs/issues/6570)

---

Ah good catch; I was taking this from https://circleci.com/docs/2.0/configuration-reference/#available-machine-images I guess is out of date.